src/interaction-handlers/games/connectFour.ts

```diff
@@ -24,10 +20,12 @@ export class ConnectFourHandler extends InteractionHandler {
   }
 
   // Get the game info and the interaction type
-  public override parse(interaction: ButtonInteraction): Maybe<{
-    gameId: number;
-    sign: number;
-  }> {
+  public override parse(interaction: ButtonInteraction):
```

Comment:
In Sapphirev4, Maybe no longer exists. We use Option.None and Option.Some instead.